streamingpro-mlsql/pom.xml

```diff
@@ -221,6 +221,11 @@
                     <artifactId>elasticsearch-spark-20_2.11</artifactId>
                     <version>6.5.0</version>
                 </dependency>
+                <dependency>
+                    <groupId>org.mongodb.spark</groupId>
+                    <artifactId>mongo-spark-connector_2.11</artifactId>
+                    <version>2.2.5</version>
+                </dependency>
```

Comment:
I think a better idea is: create a new module in external folder like `streamingpro-mongo`? so that people can chose external datasource package when they start mlsql server.
the purpose is reduce the risk of dependency conflict.